examples/brickflow_examples/workflows/demo_wf.py

```diff
@@ -296,6 +297,7 @@ def run_snowflake_queries(*args):
     sf_query_run = SnowflakeOperator(
         secret_cope="sample_scope",
         query_string="select * from table; insert into table1 select * from $database.table2",
+        #sql_file= "Path to the sql file",
```

Comment:
As operator expect one of them , can we create a different showcasing the usage .